codes/go/chapter_graph/graph_adjacency_list.go

```diff
@@ -8,31 +8,21 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
-)
-
-/* 顶点类 */
-type vertex struct {
-	val int
-}
 
-/* 构造方法 */
-func newVertex(val int) vertex {
-	return vertex{
-		val: val,
-	}
-}
+	. "github.com/krahets/hello-algo/pkg"
+)
 
 /* 基于邻接表实现的无向图类 */
 type graphAdjList struct {
 	// 邻接表，使用哈希表来代替链表，以提升删除边、删除顶点的效率
 	// 请注意，adjList 中的元素是 Vertex 对象
-	adjList map[vertex]map[vertex]struct{}
+	adjList map[Vertex]map[Vertex]struct{}
```

Comment:
We use `Map<List<Vertex>>` instead, i.e. replace the Map with List to get an ordered list.